drivers/hmis/app/graphql/types/hmis_schema/query_type.rb

```diff
@@ -171,18 +171,44 @@ def file(id:)
       Hmis::File.viewable_by(current_user).find_by(id: id)
     end
 
-    field :get_form_definition, Types::Forms::FormDefinition, 'Get most relevant/recent form definition for the specified Role and project (optionally)', null: true do
+    field :get_form_definition, Types::Forms::FormDefinition, 'Get the most relevant Form Definition to use for record viewing/editing', null: true do
       argument :role, Types::Forms::Enums::FormRole, required: true
-      argument :enrollment_id, ID, required: false
       argument :project_id, ID, required: false
     end
+    def get_form_definition(role:, project_id: nil)
```

Comment:
do you think it's worth it to rename this? Maybe `record_form_definition` ?

Reply:
sure we might as well